tools/uwuw_preproc

```diff
@@ -370,56 +401,61 @@ def copy_metadata(material, material_from_lib):
 
 
 """
-function to set the attributes of the materials:
+function to set the metadata of materials:
 ----------------------------------------
 mat : PyNE Material Object
 number : mcnp material number
-flukamat_list : 
+flukamat_list : list of materials  
 returns : PyNE Material Object
 """
+
+
 def set_metadata(mat, number, flukamat_list):
     mat.metadata['mat_number'] = str(number)
-    mat.metadata['fluka_material_index'] = str(number+25)
+    mat.metadata['fluka_material_index'] = str(number + 25)
```

Comment:
I don't remember exactly why it has to be like that. I remember I did that upon Andy's request.
@makeclean